src/concepts/config-intro.md

```diff
@@ -4,59 +4,84 @@ The Zilla runtime configuration defines the [`bindings`](../reference/config/ove
+- `remote_server` - Exists to adapt `kafka` topic streams to higher-level application streams. Read more in the [kafka-grpc binding](../reference/config/bindings/binding-kafka-grpc.md#summary).
 - `cache_client` & `cache_server` - Combined provide a persistent cache of `kafka` messages per `topic` `partition` honoring the `kafka` `topic` configuration for message expiration and compaction. Read more in the [kafka binding](../reference/config/bindings/binding-kafka.md#cache-behavior).
 
 ### Routes
 
-A Route's matching conditions are defined in terms specific to each `binding` type with some common design principles. As each incoming data stream arrives, the binding follows its configured `routes` to reach an `exit` binding or rejects the stream if no routes are viable.
+A Route's matching conditions are defined in terms specific to each `binding` type with some common design principles. As each incoming data stream arrives, the binding follows its configured `routes` to reach an `exit` binding or rejects the stream if no routes are viable. Bindings can define a default `exit` for streams that do not match any route.
 
 ::: info Order Matters
-Messages on a data stream will use the first route with a matching `when` clause in the list. Put the more specific routes first and generic routes last. Often, that looks like a wildcard route last to catch any messages that don't have a previous matching route.
+Messages on a data stream will use the first route with a matching `when` condition in the list. Order the more specific routes first and generic routes last. Often, that looks like a wildcard route last to catch any messages that don't have a previous matching route.
 :::
 
-### Route Exit
-
-A route exists to direct messages on the stream to a desired exit point.
-
 ### When a Route matches
 
-Each route can list conditions (any match) to match entry data streams. Attributes like headers, metadata, source, destination, etc., are used `when` determining the correct `exit` for a message.
-
-#### Path and Method matches
-
-Patterns for `path` and `method` routing require an exact match.
+Each route can list one or many conditions to match data streams. Attributes like headers, metadata, source, destination, etc., are used `when` determining the correct [exit](#route-exit) for the stream.
 
-- `/api/items`
-- `routeguide.RouteGuide/GetFeature`
+A route matches if any of its `when` conditions match the data stream (any match). An individual condition in a route is matched if all parts of the condition match. Meaning if multiple `when` headers are supplied for HTTP routing, then all of those headers must match the specific when condition to match.
 
-Wildcards in paths and method signatures `/*` will match multiple patterns.
+::: info Kafka topics
+Kafka topics can be defined `when` routing and `with` the route, as the destination or `reply-to` topic. In all cases, Kafka topics match exactly as defined.
```

Comment:
Suggest we change this to make it clear the `with` usage described is just an example, not the general case.

```suggestion
::: info Kafka topics
When bindings such as `http-kafka`, `grpc-kafka`, `sse-kafka` or `mqtt-kafka` route to a `kafka` binding, the route can be specified `with` Kafka topics, such as the destination or `reply-to` topic. In all cases, Kafka topics match exactly as defined.
```

Reply:
This looks good to me and makes sense. My intention here is to call out that these don't have wildcard functions unlike some other routing options. 

The part that is confusing now is in the `when` clause on the [grpc-kafka](https://docs.aklivity.io/zilla/latest/reference/config/bindings/binding-kafka-grpc.html#routes-when) binding. I don't think we allow the same kind of matching rules as with method names or wildcard topic names.

Reply:
Looking at this and more bindings I think I can just remove this specific call out and mention that individual conditions are matched exactly. then the next section can just call out the wildcard/pattern matching cases. That removes the need to discuss Kafka topics specifically

Reply:
Okay, that works too. 👍